src/services/task/routes.ts

```diff
@@ -28,18 +28,27 @@ export default [
       // req.body.payload
       // req.body.targets
       // req.body.target_regex
+      
+      let hash = crypto.createHmac('sha256', req.body.payload)
```

Comment:
move the hashing to a function to make sure its identical process with the one in repo/route